packages/frontend/src/context/wallet.tsx

```diff
@@ -101,26 +100,21 @@ const WalletProvider: React.FC = ({ children }) => {
       disconnectWallet: disconnectWallet,
       handleTransaction,
     }),
-    [web3, address, networkId, signer, balance, onWalletSelect, disconnectWallet],
+    [web3, address, networkId, signer, balance.toString()],
   )
 
```

Comment:
not able to connect my wallet from the deploy preview - is that happening for you? Everytime I click "connect wallet" I get this: 

<img width="1410" alt="Screen Shot 2022-01-22 at 2 01 56 PM" src="https://user-images.githubusercontent.com/13340486/150656821-24797edc-38ab-4a0e-a72f-42c3902fc911.png">

Reply:
Yes this is happening to me, will update